auto_dev/commands/create.py

```diff
@@ -139,7 +142,7 @@ def create(ctx, name: str, template: str, force: bool, publish: bool, clean_up:
         click.secho("Command executed successfully.", fg="yellow")
 
     if publish:
-        publish_agent(name, verbose)
+        publish_agent(public_id, verbose)
```

Comment:
So this only does the update on  publish.

Instead we need this update to occur after the initial creation